store/postgres/migrations/000001_create_tags_templates_fields_table.up.sql

```diff
@@ -0,0 +1,38 @@
+BEGIN;
+
+CREATE TABLE templates (
```

Comment:
let's rename this to `tag_templates` to make sure that this is for tagging since template is pretty general and  Columbus might have other use case for template later on.

Reply:
Sure, but in that case, I think we should also rename and do some refactor in the repository model too.